episodes/filetypes-and-metadata.md

```diff
@@ -0,0 +1,498 @@
+---
+title: 'Filetypes and metadata'
+teaching: 30
+exercises: 10
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- Which file formats should be used for microscopy images?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Explain the pros and cons of some popular image file formats
+- Explain the difference between lossless and lossy compression
+- Inspect image metadata with the napari-aicsimageio plugin
+- Inspect and set an image's scale in Napari
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Image file formats
+
+Images can be saved in a wide variety of file formats - e.g. you may be familiar 
+with some like .jpg, .png or .tiff. Microscopy images have an especially wide 
+range of options, with hundreds of different formats in use, often specific to 
+particular microscope manufacturers. With this being said, how do we choose 
+which format to use?
+
+## Metadata
+
+First, let's look closer at what gets stored inside an image file.
+
+There are two main things that get stored inside image files: _pixel values_ and 
+_metadata_. We've looked at pixel values in previous episodes -  this is the raw 
```

Comment:
Could add specific link here, i.e.
"We looked at pixel values in our episode on [multi-dimensional images](multi-dimensional-images.html)"